### PR TITLE
User service tests

### PR DESCRIPTION
**Pull request by corinnaj**

### CODE REVIEW

File: test/services/user.test.js [hunk 1]
```diff
@@ -1,10 +1,79 @@
...
+    const result = await app.service('user').get(user_id);
+
+    assert.deepEqual(result.user_id, user_id);
+    assert.include(result.achievements[0], {name: 'TestAchievement'});
+    assert.include(result.xp[0], {name: 'XP'});
+    assert.include(result.xp[0], {amount: 20});
+  });
+
+  it('does not return hidden achievements', async () => {
+    const achievement_name = 'HiddenAchievement';
```
Comment (cmfcmf) [inline test/services/user.test.js]: This variables seems a bit superfluous, given that in line 68 no variable is used.

File: test/services/user.test.js [hunk 2]
```diff
@@ -1,10 +1,79 @@
...
+
+    app.set('rules', require('../../src/rule-parser')(__dirname + '/../config/user_config.yml'));
+    app.configure(configuration());
+    app.configure(require('../../src/mongoose.js'));
+
+    await cleanDatabase(app);
+    app.configure(services);
+
+    app.service('user').setup(app);
+
```
Comment (frederike-ramin) [inline test/services/user.test.js]: remove this pleeeeaaase

File: test/services/user.test.js [hunk 3]
```diff
@@ -1,10 +1,79 @@
...
+  });
+
   it('registered the service', () => {
     const service = app.service('user');
 
     assert.ok(service, 'Registered the service');
   });
+
+  it('gives a user give xp and achievements', async () => {
+
```
Comment (frederike-ramin) [inline test/services/user.test.js]: This too pleeeeaaase

File: test/services/user.test.js [hunk 4]
```diff
@@ -1,10 +1,79 @@
...
+      name: 'TestAchievement', 
+      user_id: user_id,
+      amount: 1
+    });
+
+    const result = await app.service('user').get(user_id);
+
+    assert.deepEqual(result.user_id, user_id);
+    assert.notInclude(result.achievements[0], {name: achievement_name});
+
```
Comment (frederike-ramin) [inline test/services/user.test.js]: remoove

**Pull request merged**